alembic/versions/a1b2c3d4e5f6_add_pandascore_fields.py

```
@@ -0,0 +1,400 @@
+
+def _downgrade_match_table() -> None:
+    """Downgrade operations for the `match` table extracted from `downgrade()`.
+
+    Splitting into a focused helper reduces the apparent complexity of the
+    main `downgrade()` and makes each step easier to reason about.
+    """
+    # Drop PandaScore-specific index/columns
+    try:
+        op.drop_index(op.f("ix_match_pandascore_id"), table_name="match")
+    except Exception:
+        pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)